site-aws.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+# AWS-Specific variables and steps.
+- hosts: docker-registry
+  sudo: yes
+  vars:
+    storage_type: "s3"
+    s3_region: "eu-west-1"
+    s3_bucket: "mcp.registry.storage"
+
+  roles:
+    - aalda.docker-registry
+
+- include: common.yml
```

Comment:
Just executed a test to create a brand new docker registry, the split config fails because the tasks contained in common.yml need to be executed earlier than the docker registry config.

Problem is solved by putting the "include" line above the "docker-registry" section, please amend.

However I wonder if this current design with site-aws.yml+common.yml will cause other order of execution issues in the future.